src/main/java/org/folio/circulation/domain/representations/ItemSummaryRepresentation.java

```diff
@@ -31,8 +32,14 @@ public JsonObject createItemSummary(Item item) {
     write(itemSummary, "contributors", item.getContributorNames());
     write(itemSummary, "callNumber", item.getCallNumber());
 
-    //TODO: Check for null item status
-    writeNamedObject(itemSummary, "status", item.getStatus().getValue());
+    JsonObject status = new JsonObject()
+      .put("name", item.getStatus().getValue());
+
+    if (Objects.nonNull(item.getStatus().getDate())){
+      status.put("date", item.getStatus().getDate());
+    }
+
+    write(itemSummary, "status", status);
```

Comment:
Could this use the constant defined in `ItemProperties`?

Reply:
Yes, sure.